docker/docker-compose.test.yml

```diff
@@ -25,3 +25,12 @@ services:
       - db_test
       - critiquebrainz_redis
```

Comment:
Let's add the `musicbrainz_db` container to `links` for `critiquebrainz_test` container. I think this container starts before the `musicbrainz_db` container starts.

Reply:
Updated.